pkg/distro/BUILD

```diff
@@ -0,0 +1,68 @@
+package(
+    default_visibility = ["//visibility:private"],
+)
+
+load("@rules_pkg//:pkg.bzl", "pkg_tar")
+
+VERSION = "0.1.0"
+
+pkg_tar(
+    name = "rules_pkg-%s" % VERSION,
+    srcs = [
+        "@rules_pkg//:standard_package",
+    ],
+    extension = "tar.gz",
+    mode = "0644",
+    owner = "0.0",
```

Comment:
This is pretty opaque. What does it mean?

Reply:
Is it opaque because you are unfamiliar with RPM packaging or for different reasons?
I would do different comments depending on the reason.